core/templates/dev/head/pages/exploration_player/CurrentInteractionService.js

```diff
@@ -91,8 +85,12 @@ oppia.factory('CurrentInteractionService', [
       },
       isSubmitButtonDisabled: function() {
         /* Returns whether or not the Submit button should be disabled based on
-         * the validity of the current answer.
+         * the validity of the current answer. If there is no _validityCheckFn,
+         * return false.
```

Comment:
Could you add some more explanation to this docstring to point out that the last case is intended to be a default value?

(Otherwise, hard to get a sense of what having no _validityCheckFn actually means -- is it because it's a valid default case, or is it because the interaction takes time to load and register the _validityCheckFn, or is it an error case that shouldn't happen at all? Hard to tell.)

Reply:
Done.